certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2778 @@
+              "description": "Amount of the transaction that will be available in two business days",
+              "example": 38129.87
+            },
+            "threeDayPlusFloat": {
+              "type": "number",
+              "description": "Amount of the transaction that will be available in three or more business days",
+              "example": 78965.09
+            },
+            "MTDAverageCollected": {
+              "type": "number",
+              "description": "The amount of funds collected each day without float, divided by the number of business days in that month",
+              "example": 271840.55
+            },
+            "MTDAverageNegativeCollected": {
+              "type": "number",
+              "description": "The amount of funds negative collected each day without float, divided by the number of business days in that month",
+              "example": -640254.277
+            },
+            "totalCredits": {
+              "type": "number",
+              "description": "Sum of  previous day detail credits (BAI record type 16's)",
+              "example": 2716840.55
+            },
+            "numberOfCredits": {
+              "type": "number",
+              "description": "Item count of all previous day credits",
+              "example": 5
+            },
+            "totalDebits": {
+              "type": "number",
+              "description": "Sum of all previous day detail debits(BAI record type 16's)",
+              "example": 16840.55
+            },
+            "numberOfDebits": {
+              "type": "number",
+              "description": "Item count of all previous day debits",
+              "example": 5
+            },
+            "accessibleBalance": {
+              "type": "number",
+              "description": "Available balance plus any related Zero balance account balances plus overdraft credit amount. This balance a static balance as of the time of previous day processing, and is only available if the account is entitled.",
+              "example": 271840.55
+            },
+            "balanceAsOfDate": {
+              "type": "string",
+              "description": "ISO 8601 format",
+              "example": "2020-05-03T00:00:00.000Z"
+            }
+          }
+        }
+      ]
+    },
+    "Checks": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
+      "properties": {
+        "captureDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Capture date of the Check"
+        }
+      }
+    },
+    "lastRefreshedDateTime": {
+      "type": "string",
+      "description": "ISO 8601 Date format that the data is from.",
+      "example": "2020-05-03T00:00:00.000Z"
+    },
+    "ACH": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",

```

Comment:
@jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".